oxidation/crates/parsec_core_fs/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
crate should be put in `oxidation/libparsec/crate/core_fs` , and crate named `libparsec_core_fs` (see my comment https://github.com/Scille/parsec-cloud/pull/2334#discussion_r878904512)